pyvo/dal/tests/test_tap.py

```diff
@@ -190,6 +201,67 @@ def result(self, request, context):
         self.validator(request)
         return get_pkg_data_contents('data/tap/obscore-image.xml')
 
+    def get_job(self, request, context):
+        self.validator(request)
+        jobid = int(job_re_path.match(request.path).group(1))
+
+        job = JobFile()
+        job.jobid = jobid
+        job.phase = 'EXECUTING'
+        job.ownerid = '222'
+        job.creationtime = Time.now()
+        io = BytesIO()
+        job.to_xml(io)
+        return io.getvalue()
```

Comment:
I dig this pattern of being able to make results without hardcoded files.

Reply:
Not sure I understand...

Reply:
I just like this way of reusing the same code to make the XML.  I think it looks better than having the strings of XML, although I can understand why you'd want to do that too.  So I'm fine with either.  I think in the end, this whole test_tap thing might end up being a tap server in and of itself :wink:

No action necessary!

Reply:
Agree! I've just followed the pattern.

Reply:
> I just like this way of reusing the same code to make the XML. I think it looks better than having the strings of XML, although I can understand why you'd want to do that too. So I'm fine with either. I think in the end, this whole test_tap thing might end up being a tap server in and of itself wink
> 
> No action necessary!

It's a bit more complicated than that (ADQL Parsing), but it's a start...